modules/aws/README.md

```diff
@@ -52,7 +52,7 @@ provider "aws" {
 }
 
 module "sn_managed_cloud" {
-  source  = "streamnative/managed-cloud//modules/aws"
```

Comment:
I think this is the right syntax according to https://developer.hashicorp.com/terraform/language/modules/sources#modules-in-package-sub-directories